code/modules/integrated_electronics/subtypes/input.dm

```diff
@@ -145,6 +189,53 @@
 	push_data()
 	activate_pin(2)
 
+/obj/item/integrated_circuit/input/examiner
+	name = "examiner"
+	desc = "It' s a little machine vision system. It can return name,description,distance,relative coordinates,\
```

Comment:
`It's` [...] `It can return the name, description, distance, relative coordinates, total amount of reagents, and maximum amount of reagents of the referenced object.`